### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by saschat**

### CODE REVIEW

**Comment by saschat** [comment]
For some reason, the npm badge shows "tag not found". Not sure why.

**Comment by paddyohanlon** [comment]
> For some reason, the npm badge shows "tag not found". Not sure why.

Which npm badge?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![NPM Version](https://img.shields.io/npm/v/bzr/bazaar-js.svg?style=flat)](https://www.npmjs.com/package/@bzr/bazaar)
```
Comment (saschat) [inline README.md:1]: This NPM badge

Comment (saschat) [reply README.md:1]: Maybe it requires a `latest` tag or something. Will figure it out later... its not super important.

**Review by paddyohanlon — APPROVED** [review]

**Pull request merged**